index.html

```diff
@@ -173,6 +173,32 @@ <h2>Calls for implementation and feedback</h2>
 
 <p>When an addition is accepted at the “candidate” (stage 3) maturity level, the committee is signifying that it believes design work is complete and further refinement will require implementation experience, significant usage and external feedback.
 
+<h2>Blocking Advancement</h2>
```

Comment:
I very much don't like this phrased as "blocking" - we don't block in TC39, we simply may not provide consensus. The onus is and should be conceptually on the champion to persuade the committee to explicitly assent, not on committee members to "block".

What about:
```suggestion
<h2>Withholding Consensus for Advancement</h2>
```

Reply:
We use the phrases "I am not blocking" and "are you blocking?". We don't say "are you withholding consensus". We also do not have a way to get explicit consent from members on a proposal. We use the mechanism of silence, which is not the same, especially not in a large room where silence is the default.

I think this is a problematic part of our process, but in the interest of writing what we do down, we should write our current process rather than the ideal. At present, the onus is on committee members to block, often with a political cost. This could be observed in recent proposals, where after a block, a number of committee members voiced their unease at a proposals advancement. This is very far from explicit consent to a proposals advancement. I would be very happy if we had a way to have explicit consent on a proposal, and I think our process would benefit from it. This is a separate issue to the one we are solving here though.

Reply:
I think I sort of resolved this but please check if it fits for you.

Reply:
I agree with @codehag here, the existing practice is clearly asking for blocks?

Reply:
Historically, we asked for consensus for advancement. However, we've changed to asking if anyone objects to consensus in order to ensure everyone feels comfortable speaking up.

Certainly some folks have started wording their request as "does anyone block", but I don't think that reflects the spirit of the process document, historical behavior, or a universal agreement on the form of these things in the committee. In particular, I think the important part is that the *default* is that nothing advances, so when things don't advance, nobody is necessarily "blocking" or "obstructing progress".

Reply:
I disagree, I don't think the default velocity of proposals themselves is relevant at all to this section. This section is describing what might stop something from advancing when advancement is asked for. When advancement _is_ asked for, the default is that _unless_ someone speaks up, we assume consensus. With that default, "withholding consensus" is a mystification of what's going actively being asked: does anyone block. It'd do us a disservice to describe surfacing blocking concerns with a new term-of-art sounding language that we have not historically used.